roles/ceph-update/tasks/ceph_owner.yml

```diff
@@ -0,0 +1,23 @@
+---
```

Comment:
I think yes, ceph upgrading will not change anything until the services are restarted.
That's why we prevent ceph services restart in `ceph-common` handlers by flags